src/components/Button/__tests__/index-test.js

```diff
@@ -1,5 +1,52 @@
 /* eslint-env jasmine, jest */
 
+import React from 'react';
+import Button from '..';
+import { mount, shallow } from 'enzyme';
+
+const findNativeButton = wrapper => wrapper.find('[role="button"]');
+
 describe('components/Button', () => {
-  test.skip('NO TEST COVERAGE', () => {});
+  describe('rendered element', () => {
+    test('is a "div"', () => {
```

Comment:
This test is not necessary, as it is testing the implementation details of `Button` dependencies